components/twitter-card.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import {fallbackLocale} from './locale'
+
+function TwitterCard({
+  card = 'summary_large_image',
+  title = 'Maintainable CSS with React',
+  description = 'React component styling solved with an elegant API, small footprint, and great performance',
+  pathname,
+  locale,
+}) {
+  const urlPrefix = fallbackLocale === locale ? '' : `${locale}.`
+  const image = `https://${urlPrefix}glamorous.rocks/static/images/banner.png`
```

Comment:
We should probably also take into consideration the `rc` in the current alias for the site: https://rc.glamorous.rocks/

Currently, the preview does not work :-(

Perhaps we should do something with the `req` or `window.location` or something?